File: go/are-equally-strong/are-equally-strong.go
package main

import (
	"fmt"
	"slices"
)

func areEquallyStrong(yourLeft, yourRight, friendsLeft, friendsRight int) bool {
	myArms := []int{yourLeft, yourRight}
	friendArms := []int{friendsLeft, friendsRight}

	return slices.Max(myArms) == slices.Max(friendArms) && slices.Min(myArms) == slices.Min(friendArms) 

}

func main() {
	yourLeft := 10
	yourRight := 15
	friendsLeft := 15
	friendsRight := 10

	fmt.Printf("The answer is: %v", areEquallyStrong(yourLeft, yourRight, friendsLeft, friendsRight))
}

File: js/are-equally-strong.js
function areEquallyStrong(yourLeft, yourRight, friendsLeft, friendsRight){

    let myArms=[yourLeft,yourRight]
    let friendArms=[friendsLeft,friendsRight]
    
    let maxMyArms=Math.max(...myArms)
    let minMyArms=Math.min(...myArms)
    let maxFriendArms=Math.max(...friendArms)
    let minFriendArms=Math.min(...friendArms)
    
    return maxMyArms===maxFriendArms && minMyArms===minFriendArms
}

let yourLeft = 10, yourRight = 15, friendsLeft = 15, friendsRight = 10

console.log(areEquallyStrong(yourLeft, yourRight, friendsLeft, friendsRight))